llvm/lib/CodeGen/WinEHPrepare.cpp

```diff
@@ -51,27 +52,20 @@ static cl::opt<bool> DisableCleanups(
     cl::desc("Do not remove implausible terminators or other similar cleanups"),
     cl::init(false));
 
+// TODO: Remove this option when we fully migrate to new pass manager
```

Comment:
Sorry for the late comment; why would you want to remove this in the new pass manager? Is this option in any way related to the kind of pass manager?
(I'm asking because I added this option long ago to support Wasm EH in https://github.com/llvm/llvm-project/commit/99d60e0dabcf20f4db683da83cde905b7a1373de. Wasm EH uses WinEH IR and uses WinEHPrepare pass)

Reply:
I see, thanks.